internal/backends/winit/accesskit.rs

```diff
@@ -437,7 +437,16 @@ impl NodeCollection {
             builder.set_numeric_value_step(step);
         }
 
-        if let Some(value) = item.accessible_string_property(AccessibleStringProperty::Value) {
+        let value = item.accessible_string_property(AccessibleStringProperty::Value);
+        if value.is_none() || value.as_ref().is_some_and(|x| x.is_empty()) {
```

Comment:
I wonder if this logic shouldn't be better done in the .slint code itself.
Or is there really a reason you can't have a value and a placeholder text at the same time?

Reply:
The tests I performed with both NVDA and Narrator on Windows revealed that the placeholder would still be announced even when the widget had some text. Maybe this is due to Slint not yet implementing editable text, tests will have to be done once you get there.